### DISCUSSION TITLE
Unverified commits after changing email from gmail to protonmail

### DISCUSSION THREAD

**Discussion by learntheropes** [community/community#53585] [Repositories] [answered]
### Select Topic Area

Question

### Body

Short story:
After switching from Gmail to Github, my commits do not show as verified anymore, even if I think I followed all the required steps.

Lost story:
My GitHub account was initially registered with a Gmail address.

I was signing my commits with a pgp key associated with that Gmail, and they were showing as a verified on GitHub.

You can find an example [here](https://github.com/learntheropes/git-tests/commit/0be09c5c0ee0750e1cc770b57601f48ac714a430).

I now changed my primary GitHub email to a ProtonMail address and verified the address.

I exported from ProtonMail the private and public keys and imported them locally with `gpg --import public.key` and `gpg --import private.key` I also uploaded the pgp public key for the ProtonMail address to GitHub and ultimately trusted the keys.

On my terminal, I have down the following:
```bash
gpg --list-secret-keys --keyid-format=long
```
And I got
```bash
/home/giovanni/.gnupg/pubring.kbx
---------------------------------
sec   rsa4096/58C798100FF8A743 2023-03-17 [SC] [expires: 2039-03-13]
      5BA78A510CDA44132BDC51FA58C798100FF8A743
uid                 [ultimate] OMITTED <OMITTED@gmail.com>
uid                 [ultimate] OMITTED <giovanni@learntheropes.xyz>
uid                 [ultimate] OMITTED <OMITTED@keybase.io>
ssb   rsa4096/9EC7484EB3956638 2023-03-17 [E] [expires: 2039-03-13]

sec   rsa4096/7196D30CF74AD196 2023-04-24 [SC]
      8AB5D43DE89C0EAE9E2AA8C47196D30CF74AD196
uid                 [ultimate] giovanni.learntheropes@protonmail.ch <giovanni.learntheropes@protonmail.ch>
ssb   rsa4096/CD3CCA53A61C1DBE 2023-04-24 [E]
```
I then run:
```bash
git config --global user.signingkey 7196D30CF74AD196
git config --global commit.gpgsign true
git config --global user.email giovanni.learntheropes@protonmail.ch
```

I double-checked, and `7196D30CF74AD196` is the ID showing both on my terminal and on GitHub account next to the public key.

When I commit, I'm asked for the ProtonMail pgp password.

However, when I push the commit to GitHub remote, the commit shows as unverified (I have Vigilant mode enabled in my GitHub account),

[Here](https://github.com/learntheropes/git-tests/commit/32d2b47eb66a881e6f7183de2a7c3dea741930b0) is an example of the unverified commit with the new ProtonMail.

The message of the unverified commit says: `No user is associated with the committer email.`

What I'm doing wrong? The message to me shows as incorrect, since the commit is signed with the primary address of my Github account.

**Comment by airtower-luna** [comment]
Can you link one of the commits with the issue? Without seeing them it's hard to guess what might be wrong.

**Reply by learntheropes** [reply]
Hi, thank you for taking a look.

They are both linked in the previous message:

Verified: https://github.com/learntheropes/git-tests/commit/0be09c5c0ee0750e1cc770b57601f48ac714a430

Unverified: https://github.com/learntheropes/git-tests/commit/32d2b47eb66a881e6f7183de2a7c3dea741930b0

**Reply by airtower-luna** [reply, marked as answer]
The unverified commit isn't recognized as yours, so the error message is correct. [Looking at the details](https://github.com/learntheropes/git-tests/commit/32d2b47eb66a881e6f7183de2a7c3dea741930b0.patch) it looks like there's a typo in the author email, it's "protomail.ch" not "protonmail.ch".

**Reply by learntheropes** [reply]
Damn! Thanks a lot. At least I learned from you to add `.patch` at the end of the URL to see the details.
